src/Entity/OgRole.php

```diff
@@ -38,138 +34,135 @@
 class OgRole extends Role implements OgRoleInterface {
 
   /**
-   * @var integer
-   *
-   * The group ID.
-   */
-  protected $group_id;
-
-  /**
-   * The entity type ID of the group.
-   *
-   * @var string
-   */
-  protected $group_type;
-
-  /**
-   * The bundle ID of the group.
-   *
-   * @var string
-   */
-  protected $group_bundle;
-
-  /**
    * Set the ID of the role.
    *
    * @param string $id
    *   The machine name of the role.
    *
-   * @return OgRole
+   * @return $this
```

Comment:
Good question. I changed it to $this because it satisfies PHP CodeSniffer. I will see there is a mention for documentation of chainable methods in the OO coding standards. I'll also have a look at how this is usually done in Drupal core.